x-pack/plugins/enterprise_search/public/applications/app_search/components/search_ui/utils.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import queryString, { ParsedQuery } from 'query-string';
+
+import { getAppSearchUrl } from '../../../shared/enterprise_search_url';
+
+export const generatePreviwUrl = (query: ParsedQuery, engineName: string) => {
```

Comment:
Super sorry to do this, but there's a small typo in this fn name 😬 

```suggestion
export const generatePreviewUrl = (query: ParsedQuery, engineName: string) => {
```

Reply:
Super appreciate it being a separate util/file though!!

Also just thinking out loud, I wonder if we should bake the `fromKibana: true` flag into this helper vs passing it in manually in the logic file. If we anticipate being able to delete `fromKibana` in 8.0 it kind of makes it nice and simple to delete it in 1 single place/file rather than multiple.

Reply:
I totally noticed that but forgot to circle back and fix it.

I'm fine with that. I have to make updates anyway so I will switch that around.